docs/src/examples/jump_umap_analysis_with_cosmicqc.py

```diff
@@ -0,0 +1,421 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.16.2
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# # JUMP UMAP analysis with coSMicQC
+#
+# This notebook analyzes [JUMP](https://jump-cellpainting.broadinstitute.org/) data by leveraging [UMAP](https://arxiv.org/abs/1802.03426) and [coSMicQC](https://github.com/WayScience/coSMicQC).
+#
+# ## Outline
+#
+# We focus on a single file from the JUMP dataset: [`BR00117012.sqlite`](https://open.quiltdata.com/b/cellpainting-gallery/tree/cpg0000-jump-pilot/source_4/workspace/backend/2020_11_04_CPJUMP1/BR00117012/BR00117012.sqlite).
+# This file is downloaded and prepared by [CytoTable](https://github.com/cytomining/CytoTable) to form a single-cell [Parquet](https://parquet.apache.org/) file which includes all compartment feature data.
+# We use coSMicQC to find and remove erroneous outlier data in order to prepare for UMAP analysis.
+# Afterwards, we use UMAP to demonstrate patterns within the data.
+#
+
+# + editable=true slideshow={"slide_type": ""}
+import logging
+import pathlib
+from typing import List, Optional
+
+import cosmicqc
+import holoviews
+import hvplot.pandas
+import numpy as np
+import pandas as pd
+import pycytominer
+import umap
+from cytotable.convert import convert
+from parsl.config import Config
+from parsl.executors import ThreadPoolExecutor
+from pyarrow import parquet
+
+# set bokeh for visualizations with hvplot
+hvplot.extension("bokeh")
+
+# create a dir for images
+pathlib.Path("./images").mkdir(exist_ok=True)
+
+# avoid displaying plot export warnings
+logging.getLogger("bokeh.io.export").setLevel(logging.ERROR)
+
+# +
+# check if we already have prepared data
+if not pathlib.Path("./BR00117012.parquet").is_file():
+    # process BR00117012.sqlite using CytoTable to prepare data
+    merged_single_cells = convert(
+        source_path=(
+            "s3://cellpainting-gallery/cpg0000-jump-pilot/source_4/workspace"
+            "/backend/2020_11_04_CPJUMP1/BR00117012/BR00117012.sqlite"
+        ),
+        dest_path="./BR00117012.parquet",
+        dest_datatype="parquet",
+        source_datatype="sqlite",
+        chunk_size=8000,
+        preset="cellprofiler_sqlite_cpg0016_jump",
+        no_sign_request=True,
```

Comment:
Recommend adding code comment on with no_sign_request is.

Reply:
Thanks - I'll add in a comment about this.